src/ucp/wireup/select.c

```diff
@@ -2125,9 +2125,11 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
         bw_info.max_lanes = context->config.ext.max_rndv_lanes;
     }
 
-    /* If error handling is requested we require memory invalidation
-     * support to provide correct data integrity in case of error */
-    if (ep_init_flags & UCP_EP_INIT_ERR_MODE_PEER_FAILURE) {
+    /* If error handling is requested and we have RNDV, we require memory
+     * invalidation support to provide correct data integrity in case of error.
+     */
+    if ((ep_init_flags & UCP_EP_INIT_ERR_MODE_PEER_FAILURE) &&
+        ucp_context_rndv_is_enabled(context)) {
```

Comment:
I recommend adding a gtest with enabled/disabled rndv via setting rndv threshold, checking the behavior of ep created using `UCP_EP_INIT_ERR_MODE_PEER_FAILURE`.

Reply:
i agree, but since this PR is urgent, @tvegas1 can you pls squash and implement gtest in the next PR